tests/connections.py

```diff
@@ -23,7 +23,7 @@
 
 
 def run(args):
-    hosts = ["localhost"]
+    hosts = ["localhost"] * (4 if args.consensus == "pbft" else 1)
```

Comment:
Shouldn't this work with one node in PBFT (`f = 0`)?

Reply:
if the network is openend f becomes 1 though